### PR TITLE
70527: Rework Storage > Multipaths text

### PR DESCRIPTION
**Pull request by Mrt134**
HTML build test: no issues.
Port text improvements to master

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -493,6 +493,11 @@ U2
 U3
 ~~
 
+* When compatible hardware is connected, the :guilabel:`Multipaths` page
```
Comment (a-stjohn) [inline userguide/intro.rst]: I'm a little confused with this sentence. It seems to be a typo. Or perhaps 2 sentences?
From what I understand, it appears that the 'Multipaths' page has been added in U3, but it only appears when compatible hardware is connected. So, perhaps:

`A :guilabel:`Multipaths`page now appears in the
:menuselection:`Storage`
menu when compatible hardware is connected.`

Comment (Mrt134) [reply userguide/intro.rst]: I tried to put the caveat about compatible hardware first, but I'll rework it. Thanks!

File: userguide/storage.rst [hunk 2]
```diff
@@ -1270,30 +1269,31 @@ should not be used with :ref:`Windows (SMB) Shares`.
...
+hardware.
 
 %brand% uses
 `gmultipath(8) <https://www.freebsd.org/cgi/man.cgi?query=gmultipath>`__
 to provide
 `multipath I/O <https://en.wikipedia.org/wiki/Multipath_I/O>`__
-support on systems containing hardware that is capable of multipath.
-An example would be a dual SAS expander backplane in the chassis or an
-external JBOD.
+support on systems containing multipath-capable hardware like a chassis
```
Comment (wblock) [inline userguide/storage.rst]: Move the examples of multipath hardware to the introductory paragraph.

File: userguide/storage.rst [hunk 3]
```diff
@@ -1270,30 +1269,31 @@ should not be used with :ref:`Windows (SMB) Shares`.
...
+equipped with a dual SAS expander backplane, or an external JBOD that is
```
Comment (wblock) [inline userguide/storage.rst]: No comma needed here.

**Pull request merged**